test/unit/test_job_list.py

```diff
@@ -334,8 +334,10 @@ def test_that_create_method_makes_the_correct_calls(self):
         job_list._add_dependencies.assert_called_once_with(date_list, member_list, chunk_list, cj_args[0])
         # Adding flag update structure
         job_list.update_genealogy.assert_called_once_with()
-        for job in job_list._job_list:
-            self.assertEqual(parameters, job.parameters)
+
+        # job doesn't have job.parameters anymore TODO
+        # for job in job_list._job_list:
+        #     self.assertEqual(parameters, job.parameters)
```

Comment:
I think these can be deleted? Or will we be able to test it later in some other way?

Reply:
Then I think it'd be a good idea to add a TODO or a GitHub issue so we don't forget about it.